aws/cloudformation/components/ami.yml.erb

```diff
@@ -112,11 +118,12 @@ frontend_device_name ||= '/dev/sda1'
     LatestStackUpdateDateTime: Time.now # Trigger update of this custom Resource (invocation of the Lambda) for each Stack update.
   %>
   Frontends:
+    DependsOn: [ASGCount]
```

Comment:
Any `GetAtt` or `Ref` calls to another resource implicitly create a `DependsOn` relationship, so you don't need to explicitly list it here.